REQUIREMENTS.txt

```diff
@@ -9,6 +9,7 @@ semantic_version
 scikit-image>=0.14.0
 scikit-learn
 scipy
+seaborn
```

Comment:
add to requirements-notebook?

Reply:
Oh I thumbs-upped too soon. I think it needs to be in dev because starfish will die if this doesn't get installed, since it gets imported in the vis suite.

Reply:
What's the rule for something that belongs in requirements-notebook?